templates/zerver/meta_tags.html

```diff
@@ -0,0 +1,16 @@
+<!-- Google / Search Engine Tags -->
+<meta name="description" content="Zulip is powerful, open source team chat. Zulip combines the immediacy of real-time chat with the productivity benefits of threaded conversations.">
+
+<!-- Facebook Meta Tags -->
+<meta property="og:url" content="https://zulipchat.com">
```

Comment:
Since folks can run their own Zulip server, this should be `{{ realm_uri }}` to show the domain for the server in question.  I'll just change this and merge.